src/Backend/HPTA.Api/appsettings.json

```diff
@@ -23,5 +23,8 @@
   "MicrosoftGraph": {
     "BaseUrl": "https://graph.microsoft.com/v1.0",
     "Scopes": "user.read"
+  },
+  "AppSettings": {
```

Comment:
It is better to name the property as "ConnectionStrings" instead of "AppSettings". We will add more properties into the app settings and will use Options pattern to access them.

Reply:
ConnectionStrings is usually used for any server related connection strings right? AppSettings refer to any application related settings. We can add more properties here as well and  we will use options pattern to access them

Reply:
There is a built-in implementation to access connection string in IConfiguration interface (ex: configuration.GetConnectionString("Name of connection string")). If we enclose our connection string inside any other property ("AppSettings" in our case), the default implementation breaks. There are many ORMs which access connection string from IConfiguration. Hence I would suggest getting connection string using built-in implementation which requires connection string to be enclosed in "ConnectionStrings" property.